src/server/frappy/frappapi/views.py

```diff
@@ -128,6 +128,17 @@ def perform_create(self, serializer, cost):
             final_price=cost,
         )
 
+    @action(detail=True, methods=["get", "post"])
```

Comment:
is this still needed? Or if it is, do we still keep the `frappe.status = not frappe.status` line?